translations/ru_RU.ts

```diff
@@ -1,966 +1,973 @@
+        <translation>Принудительный выход</translation>
     </message>
     <message>
         <source>In the last session, CP Editor was abnormally killed, do you want to restore the last session?</source>
-        <translation type="unfinished"></translation>
+        <translation>Предыдущая сессия CP Editor была завершена некорректно. Вы хотите восстановить предыдущую сессию?</translation>
     </message>
     <message>
         <source>Show Main Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Показать Главное окно</translation>
     </message>
     <message>
         <source>Opening Files</source>
-        <translation type="unfinished"></translation>
+        <translation>Открытие файлов</translation>
     </message>
     <message>
         <source>https://github.com/cpeditor/cpeditor/blob/%1/doc/MANUAL.md</source>
-        <translation type="unfinished"></translation>
+        <translation>https://github.com/cpeditor/cpeditor/blob/%1/doc/MANUAL_ru-RU.md</translation>
     </message>
     <message>
         <source>About CP Editor %1</source>
-        <translation type="unfinished"></translation>
+        <translation>О CP Editor %1</translation>
     </message>
     <message>
         <source>&lt;p&gt;&lt;b&gt;CP Editor&lt;/b&gt; is a native Qt-based code editor. It&apos;s specially designed for competitive programming, unlike other editors/IDEs which are mainly for developers. It helps you focus on your algorithm and automates the compilation, executing and testing. It even fetches test cases for you from different platforms and submits solutions to Codeforces!&lt;/p&gt;&lt;p&gt;Copyright (C) 2019-2020 Ashar Khan &amp;lt;ashar786khan@gmail.com&amp;gt;&lt;/p&gt;&lt;p&gt;This is free software; see the source for copying conditions. There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. The source code for CP Editor is available at &lt;a href=&quot;https://github.com/cpeditor/cpeditor&quot;&gt; https://github.com/cpeditor/cpeditor&lt;/a&gt;.&lt;/p&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;p&gt;&lt;b&gt;CP Editor&lt;/b&gt; - редактор кода, созданный на Qt. Дизайн CP Editor специально рассчитан на спортивное программирование, в отличии от других редакторов/IDE, который больше подходят разработчикам. Наш редакатор кода помогает сфокусироваться на построении алгоритма, автодополняя, запуская и тестируя его. Он также может использовать тест-кейсы разных платформ и отправлять решения на Codeforces!&lt;/p&gt;&lt;p&gt;Copyright (C) 2019-2020 Ashar Khan &amp;lt;ashar786khan@gmail.com&amp;gt;&lt;/p&gt;&lt;p&gt;Это бесплатное ПО; см. источник исходного кода. АВТОР НЕ ДАЕТ НИКАКИХ ГАРАНТИЙ; ни для пригодности для использования, ни для личных целей. Исходный код CP Editor доступен на &lt;a href=&quot;https://github.com/cpeditor/cpeditor&quot;&gt; https://github.com/cpeditor/cpeditor&lt;/a&gt;.&lt;/p&gt;</translation>
```

Comment:
```suggestion
        <translation>&lt;p&gt;&lt;b&gt;CP Editor&lt;/b&gt; - редактор кода, созданный на Qt. Дизайн CP Editor специально рассчитан на спортивное программирование, в отличии от других редакторов/IDE, который больше подходят разработчикам. Наш редакатор кода помогает сфокусироваться на построении алгоритма, автодополняя, запуская и тестируя его. Он также может использовать тест-кейсы разных платформ и отправлять решения на Codeforces!&lt;/p&gt;&lt;p&gt;Copyright (C) 2019-2020 Ashar Khan &amp;lt;ashar786khan@gmail.com&amp;gt;&lt;/p&gt;&lt;p&gt;Это свободно распространяемое программное обеспечение. Условия копирования приведены в исходных текстах. Без гарантии каких-либо качеств, включая коммерческую ценность и применимость для каких-либо целей. Исходный код CP Editor доступен на &lt;a href=&quot;https://github.com/cpeditor/cpeditor&quot;&gt; https://github.com/cpeditor/cpeditor&lt;/a&gt;.&lt;/p&gt;</translation>
```

This is copied from `LANGUAGE=ru_RU g++ --version`. (The source text is copied from `LANG=C g++ --version`.)